### PR TITLE
Layout music player

### PR DESCRIPTION
**Pull request by imartinez921**
Uses flex containers to implement Flexbox layout within music player in footer.
Adds music player buttons.
Adds box-sizing CSS to * elements for clearer layout.
Fixes ArtistShow background and PageContainer background from shrinking with display size.

_Before:_

https://github.com/imartinez921/versify_full-stack/assets/102888592/18fa054e-3539-42a8-92ba-452d8df00c04

_Result:_

https://github.com/imartinez921/versify_full-stack/assets/102888592/b88bf63b-63e4-48a7-b77b-9f24be6f1ad2

### CODE REVIEW

File: app/assets/stylesheets/00-css-reset.css [hunk 1]
```diff
@@ -46,4 +46,8 @@ q:before, q:after {
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
+}
+
+* {
```
Comment (k-shan) [inline app/assets/stylesheets/00-css-reset.css:51]: What does * do? Also I'm curious why your css files are numbered?

File: frontend/components/footer/footer_container.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Connect } from "react-redux";
+import { connect } from "react-redux";
```
Comment (k-shan) [inline frontend/components/footer/footer_container.js:1]: Is this change intended?

**Review by k-shan — APPROVED** [review]

File: app/assets/stylesheets/00-css-reset.css (hunk 1 above)
Comment (imartinez921) [reply app/assets/stylesheets/00-css-reset.css:51]: `*` selects for all elements. It was recommended to add box-sizing to everything to make layout easier because it will case element width to include border and padding.

My CSS files are numbered because old Rails uses a CSS compiler, so we were originally taught to separate the CSS files by component for organization, but the numbered filenames keep the files in sequence since CSS code in a later file could override previous code, if that makes sense.

File: frontend/components/footer/footer_container.js (hunk 2 above)
Comment (imartinez921) [reply frontend/components/footer/footer_container.js:1]: Yes, the function isn't capitalized.
